### ISSUE TITLE
PHP 7: Symphony Warning: Declaration of fieldMultilingual_TextBox::getCurrentHandle($entry_id, $lc) should be compatible with FieldTextBox::getCurrentHandle($entry_id, $lc = NULL)

### ISSUE THREAD

**Issue by wdebusschere** [DeuxHuitHuit/multilingual_field#79]
````
An error occurred in /Users/xxx/Sites/www.xxx.org/extensions/multilingual_field/fields/field.multilingual_textbox.php around line 10
5 }
6
7 require_once(EXTENSIONS . '/frontend_localisation/lib/class.FLang.php');
8 require_once(EXTENSIONS . '/textboxfield/fields/field.textbox.php');
9
10 class fieldMultilingual_TextBox extends FieldTextBox
11 {
12 /*------------------------------------------------------------------------------------------------*/
13 /* Definition */
14 /*------------------------------------------------------------------------------------------------*/

**Comment by nitriques** [comment]
Can you tell me which versions of both extension you are running ?

**Comment by wdebusschere** [comment]
frontend_localisation 2.7.3
textboxfield 2.7.3

**Comment by nitriques** [comment]
wow they are the same ;)

And I guess you are running the latest multilingual_field, i.e. 3.5.13 ?

Can you test #75 please ?

**Comment by wdebusschere** [comment]
Yes, that worked

**Comment by nitriques** [comment]
Good. Released as `3.5.14`
